js/TableDetailComponent.js

```diff
@@ -56,8 +56,15 @@ class TableDetailComponent {
             event.preventDefault();
             this.closeDetail();
         });
-    }
 
+        //Close with backdrop click
+        this.$html.find('.table-detail-backdrop').on('click', (event) => {
+            event.preventDefault();
+            if (this.$html.find('.table-detail-backdrop').hasClass('in')) {
+                this.closeDetail();
+            }
+        });
```

Comment:
Does the `'.table-detail-backdrop'` ever get removed from the DOM? If not could we create a variable and then reference it inside the event handler rather than perform the additional `find(...)`.

If we do call `remove(...)` on it, could we instead use `detach(...)` (https://api.jquery.com/detach/)